docs/running_docker.md

```diff
@@ -45,55 +45,69 @@ For example, to run readout, modify the `_plain_command` section of [`readout.ya
 When running readout, we successfully used the following command inside the `alma9-flp-node` image:
 
 ```bash
-sudo /usr/bin/docker run --name readout --replace \
-  --user "$(id -u flp):$(id -u flp)" \
+sudo -E docker run --name readout --replace \
+  --user flp -v /etc/group:/etc/group:ro -v /etc/passwd:/etc/passwd:ro \
+  --privileged \
   --network=host --ipc=host \
-  -e O2_DETECTOR -e O2_PARTITION -e OCC_CONTROL_PORT \
-  -e O2_SYSTEM -e O2_ROLE \
+  -v /tmp:/tmp \
+  -v /lib/modules/$(uname -r):/lib/modules/$(uname -r) \
+  -e O2_DETECTOR -e O2_PARTITION -e OCC_CONTROL_PORT -e O2_SYSTEM -e O2_ROLE \
   gitlab-registry.cern.ch/aliceo2group/dockerfiles/alma9-flp-node:2 \
   /opt/o2/bin/o2-readout-exe
```

Comment:
perhaps annotate with a comment the bits which are needed only for running with a readout card.